examples/c_json.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import print_function
```

Comment:
Mind adding a short blurb explaining what this example does and how to use it, at the top? Similar to other examples...